app/services/catalog/update_order_item.rb

```diff
@@ -0,0 +1,59 @@
+module Catalog
+  class UpdateOrderItem
+    class OrderItemNotFound < StandardError; end
+    class ServiceInstanceNotFound < StandardError; end
+
+    def initialize(topic)
+      @payload = topic.payload
+      @message = topic.message
+    end
+
+    def process
+      Rails.logger.info("Processing service order topic message: #{@message} with payload: #{@payload}")
+
+      Rails.logger.info("Searching for OrderItem with a task_ref: #{@payload["task_id"]}")
+      order_item = find_order_item
+      Rails.logger.info("Found OrderItem: #{order_item.id}")
+
```

Comment:
@eclarizio We need to set the context here akin to this
https://github.com/ManageIQ/catalog-api/blob/b1d58844bad7321ff7825a490bdc757a306a23be/app/services/catalog/order_item_transition.rb#L13

This will ensure that your logging as well as tenancy related stuff works seamlessly

Reply:
Ah yeah, I forgot to push that up on the other PR first. Fixed there and will rebase here.